packages/design-system/src/components/OcBreadcrumb/OcBreadcrumb.vue

```diff
@@ -200,6 +216,18 @@ export default defineComponent({
       return document.querySelector(`.oc-breadcrumb-list [data-item-id="${id}"]`)
     }
 
+    const dragOver = (event) => {
+      event.preventDefault()
+    }
+
+    const dropItemEvent = (item, index) => {
```

Comment:
You also need to forbid using the `Spaces`  overview or `Shared with me` overview. Super ugly, but one possible solution could be to check if `index === (props.transactionOffset - 2)`... but would prefer if you had a better idea for this.

Reply:
hm maybe we could make it a isTruncationPlaceholder, then it should also be disabled to drop there (and the logic doesnt change much for hover and drop)